server/src/test/java/server/GameControllerTest.java

```diff
@@ -69,14 +78,16 @@ void checkBulletsData() {
 	void checkIfJSONArrayOfBulletsIsNotEmpty() {
 		JSONArray bulletsData = gameController.getBulletsData();
 		
-		assertFalse(bulletsData.isEmpty());
+		assertThat(bulletsData.isEmpty()).isFalse();
 	}
 	
 	@Test
 	void checkIfJSONArrayHasTheSameSizeLikeListOfBullets() {
-		assertEquals(getPreparedBullets().size(), gameController.getBulletsData().length());
+		assertThat(getPreparedBullets().size()).isEqualTo(gameController.getBulletsData().length());
```

Comment:
The argument of _assertThat_ should be component that you are supposed to test, not prepared data. Please change order of the assertion.